pandas/tests/frame/test_arithmetic.py

```diff
@@ -931,16 +931,16 @@ def test_binop_other(self, op, value, dtype):
         elif (op, dtype) in skip:
 
             if op in [operator.add, operator.mul]:
-                with tm.assert_produces_warning(UserWarning):
-                    # "evaluating in Python space because ..."
-                    op(s, e.value)
+                # TODO we should assert this or not depending on whether
+                # numexpr is used or not
+                # with tm.assert_produces_warning(UserWarning):
+                #     # "evaluating in Python space because ..."
+                op(s, e.value)
```

Comment:
The fact that it is no longer warning is I *think* correct, since we are not using numexpr for such a small test case (so the warning about "not supported by numexpr" is not relevant for the user). 
But, we should still test that we *do* raise the warning when numexpr is used. This can probably be done in a similar was as https://github.com/pandas-dev/pandas/pull/40463

Reply:
Will take care of this in https://github.com/pandas-dev/pandas/pull/41178